tests/by-util/test_tail.rs

```diff
@@ -266,8 +241,12 @@ fn test_follow_redirect_stdin_name_retry() {
 }
 
 #[test]
-#[cfg(not(target_os = "macos"))] // See test_stdin_redirect_dir_when_target_os_is_macos
-#[cfg(all(unix, not(any(target_os = "android", target_os = "freebsd"))))] // FIXME: fix this test for Android/FreeBSD
+#[cfg(all(
+    not(target_vendor = "apple"),
```

Comment:
Not really a comment, more a question. Why are you using `target_vendor` instead of `target_os`? One includes iOS and the other one doesn't?

Reply:
I looked through the repo and it seems like this is the preferred method to switch off tests for macos or apple in general. We only test on macos in the CI, but most probably this test won't work on iOS or something else from apple.